htdocs/core/lib/functions.lib.php

```diff
@@ -51,7 +51,7 @@ function getDolGlobalString($key)
 {
 	global $conf;
 	// return $conf->global->$key ?? '';
-	return (string) (empty($conf->global->$key) ? '' : $conf->global->$key);
+	return (string) (isset($conf->global->$key) ? $conf->global->$key : '');
```

Comment:
Hello, this is already fixed since v18; i dont know if @eldy still apply fixes on v14.